lib/api.js

```diff
@@ -21,7 +21,7 @@ export default {
     },
     /**
      * match expression for of his search in nodes of tree
-     * @param  {*}        expression    Object/String... for search. Array is an enumeration expressions
+     * @param  {*}        expression    Object|String|RegExp for search. Array is an enumeration expressions
```

Comment:
`@param {String|RegExp|Object|Array.<String|RegExp|Object>} expression - Matcher(s) to search`